src/pages/login/CoreConnect.tsx

```diff
@@ -65,8 +69,47 @@ const CoreConnect = () => {
       });
   };
 
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch('http://132.145.101.179:42069/');
+        if (!response.ok) {
+          setShowPopup(true);
+        }
+        else {
+        }
+      }
+      catch (error) {
+        setShowPopup(true);
+        console.log("fail");
+      }
+    };
+
+    fetchData();
+  }, []);
+
   return (
     <div className="flex w-[768px] max-w-full flex-col items-stretch justify-center gap-12 rounded-2xl px-12 py-14 @container">
+      {showPopup && (
+        <div>
+          <ConfirmDialog
+            isOpen={showPopup}
+            onClose={() => setShowPopup(false)}
+            title="HTTP Error"
+            type="danger"
+          >
+            You may need to adjust your network and browser settings.{' '}
```

Comment:
Since we already know that it's an http/mixed content issue, at should just say it explicitly.
Something like "Missing permission: mixed content, we can't load http content over https if you don't enable this" but write a better sentence